src/app/api/api_v1/endpoints/organizations.py

```diff
@@ -27,7 +32,12 @@ async def register_organization(
     telemetry_client.capture(
         token_payload.sub, event="organization-create", properties={"organization_name": payload.name}
     )
-    return await organizations.create(payload)
+    organization = await organizations.create(payload)
```

Comment:
Mmmh I was about to suggest we do this after making sure the S3 ops haven't failed, but we need the org ID for the bucket :thinking:

Reply:
we can catch Error from the S3 ops and return an error ?